DragaliaAPI.Shared/MasterAsset/Models/QuestDrops/QuestBonusReward.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Text.Json.Serialization;
+using DragaliaAPI.Shared.Definitions.Enums;
+
+namespace DragaliaAPI.Shared.MasterAsset.Models.QuestDrops;
+
+public record QuestBonusReward(int QuestId, IEnumerable<Drop> Bonuses);
+
+// TODO: Extend with random quantity variation
+public record Drop(
```

Comment:
Maybe rename this? so its not as generic as just "Drop", maybe "QuestBonusDrop" or "QuestBonusEntity"